Sources/WishKit/Configuration/Config+Buttons.swift

```diff
@@ -16,5 +16,7 @@ extension Configuration {
         public var voteButton = VoteButton()
 
         public var saveButton = SaveButton()
+
+        public var cancelButton = SaveButton()
```

Comment:
I assume the name `SaveButton` only describes the Look and not the behavior of the button?